client/Makefile

```diff
@@ -3,7 +3,7 @@ ldflags="-X github.com/katzenpost/katzenpost/core/epochtime.WarpedEpoch=${warped
 uid=$(shell [ "$$SUDO_UID" != "" ] && echo "$$SUDO_UID" || id -u)
 gid=$(shell [ "$$SUDO_GID" != "" ] && echo "$$SUDO_GID" || id -g)
 docker_user=${uid}:${gid}
-docker_args=--user ${docker_user} -v $(shell readlink -f ..):/go/katzenpost -e GOCACHE=/tmp/gocache --network=host --ulimit memlock=1048576 --rm -it
+docker_args=--user ${docker_user} -v $(shell readlink -f ..):/go/katzenpost -e GOCACHE=/tmp/gocache --network=host --ulimit memlock=1048576 --rm -it --volume /usr/local/include/sphincsplus:/usr/local/include/sphincsplus
```

Comment:
we should probably build sphincsplus inside the docker?

Reply:
why? Currently we'll need to build sphincsplus in order to run many of the unit tests.

Reply:
Hmm, I didn't realize we didn't do the entire build inside Docker. In that case, the `--volume` will go away when `sphincsplus.o` gets linked into the binaries by `cgo` at buildtime.